#!/usr/bin/env python
PACKAGE = "lowlevel_ctrl"
import roslib; roslib.load_manifest(PACKAGE)
from math import pi
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("enforce_throttle", bool_t, 0, "enable_throttle", False)
gen.add("set_goal",   bool_t,   0, "set_goal",  False)
gen.add("goal_x", double_t, 0, "goal_x",    13.5, -25,   25)
gen.add("goal_y", double_t, 0, "goal_y",    27.79, -25,   25)


exit(gen.generate(PACKAGE, "lowlevel_ctrl", "test"))